app/main/views/frameworks.py

```diff
@@ -273,6 +273,73 @@ def framework_start_supplier_declaration(framework_slug):
                            framework_close_date=framework_close_date), 200
 
 
+@main.route('/frameworks/<framework_slug>/declaration/reuse', methods=['GET'])
+@login_required
+def reuse_framework_supplier_declaration(framework_slug):
+    """Attempt to find a supplier framework declaration that we can reuse.
+
+    :param: framework_slug
+    :query_param: reusable_declaration_framework_slug
+    :return: 404, redirect or reuse page (frameworks/reuse_declaration.html)
+    """
+    reusable_declaration_framework_slug = request.args.get('reusable_declaration_framework_slug', None)
+    supplier_id = current_user.supplier_id
+    # Get the current framework.
+    current_framework = data_api_client.get_framework(framework_slug)['frameworks']
+    if reusable_declaration_framework_slug:
+        # If they have supplied a declaration then force it. This is an 'admin' feature.
+        # Experimental feature to be used to force through reuse.
```

Comment:
Shouldn't be "Experimental" - that implies it might not work :)

I'd go with something like 
```
# If a framework slug is supplied in the URL then use this for pre-filling if possible,
# overriding the default framework selection logic.
```